napari_allencell_annotator/view/viewer.py

```diff
@@ -1,30 +1,59 @@
-from typing import List
+from typing import List, Tuple, Dict, Optional
+from enum import Enum
 
+import dask.array
 import numpy as np
-from napari.layers import Layer
-
+from bioio import BioImage
+from napari.layers import Layer, Points
 from napari_allencell_annotator.view.i_viewer import IViewer
 from napari.utils.notifications import show_info
 import napari
 
 
+class PointsLayerMode(Enum):
+    """
+    Mode for view.
+
+    ADD is used to add points.
+    SELECT is used to move, edit, or delete points.
+    PAN_ZOOM is the default mode and allows normal interactivity with the canvas.
+    """
+
+    ADD = "add"
+    SELECT = "select"
+    PAN_ZOOM = "pan_zoom"
+
+
 class Viewer(IViewer):
     """Handles actions related to napari viewer"""
 
     def __init__(self, viewer: napari.Viewer):
         super().__init__()
         self.viewer: napari.Viewer = viewer
 
-    def add_image(self, image: np.ndarray) -> None:
+    def add_image(self, image: BioImage) -> None:
         """
         Add an image to the napari viewer
 
         Parameters
         ----------
-        image: np.ndarray
+        image: BioImage
             An image to be added
         """
-        self.viewer.add_image(image)
+        # layer: Optional[napari.layers.Layer] = None
+        # # For multiscene images
+        # if len(image.scenes) > 0:
+        #     for i in range(len(image.scenes)):
+        #         # add each scene separately
+        #         data: dask.array.Array = image.get_image_dask_data(image.dims.order.replace("S", ""), S=i)
+        #         layer = self.viewer.add(data)
+        # else:
+        # # for all other images <=5 dims
+        #     layer = self.viewer.add(image.get_dask_stack())
+
+        self.viewer.add_image(image.get_dask_stack())
+
+        # layer.axis_labels = image.dims.order
```

Comment:
this should take in a numpy array

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/5a1eac3472eb10cd133a20dd0b9a299845829cba